### PR TITLE
chore: update trengo drip kustomer axios handler

### PR DESCRIPTION
**Pull request by ItsSudip**
## What are the changes introduced in this PR?

Write a brief explainer on your code changes.

## What is the related Linear task?

Resolves INT-1910

## Please explain the objectives of your changes below

Put down any required details on the broader aspect of your changes. If there are any dependent changes, **mandatorily** mention them here

### Any changes to existing capabilities/behaviour, mention the reason & what are the changes ?

N/A

### Any new dependencies introduced with this change?

N/A

### Any new generic utility introduced or modified. Please explain the changes.

N/A

### Any technical or performance related pointers to consider with the change?

N/A

@coderabbitai review

<hr>

### Developer checklist

- [ ] My code follows the style guidelines of this project

- [ ] **No breaking changes are being introduced.**

- [ ] All related docs linked with the PR?

- [ ] All changes manually tested?

- [ ] Any documentation changes needed with this change?

- [ ] Is the PR limited to 10 file changes?

- [ ] Is the PR limited to one linear task?

- [ ] Are relevant unit and component test-cases added in **new readability format**?

### Reviewer checklist

- [ ] Is the type of change in the PR title appropriate as per the changes?

- [ ] Verified that there are no credentials or confidential data exposed with the changes.

### CODE REVIEW

**Comment by devops-github-rudderstack** [comment]
Test report for this run is available at: https://test-integrations-dev.s3.amazonaws.com/integrations-test-reports/rudder-transformer/3488/test-report.html

File: src/v0/destinations/drip/util.js [hunk 1]
```diff
@@ -77,18 +82,21 @@ const createUpdateUser = async (finalpayload, Config, basicAuth) => {
...
         module: 'router',
       },
     );
+    const { response } = httpResponse;
     if (response) {
       return response.status === 200 || response.status === 201;
     }
     throw new AbortedError('Invalid response.');
-  } catch (error) {
+  } catch ({ destinationResponse }) {
```
Comment (sanpj2292) [inline src/v0/destinations/drip/util.js:98]: Interesting... Are we confident that `destinationResponse` will always be present ?

Comment (sanpj2292) [reply src/v0/destinations/drip/util.js:98]: In `myAxios` as soon as we figure out there is an error status-code, we would throw an exception. May be that is the reason we were having a catch block in the first-place.
But now with `handleHttpRequest` method, we wouldn't need to have a catch block may be. Instead we should rely on the response that we got from `handleHttpRequest` & throw an error when required.

May be we should re-think existence of catch block ?

**Review by sanpj2292 — APPROVED** [review]

**Comment by aashishmalik** [comment]
@ItsSudip no test files change for drip, kustomer?

**Comment by ItsSudip** [comment]
> @ItsSudip no test files change for drip, kustomer?

Yes. As there is no change in the functionality, there is no test file change.

**Review by aashishmalik — APPROVED** [review]

**Pull request merged**